version/fabricversion.go

```diff
@@ -44,6 +44,7 @@ const (
 	V2_2_5 = "2.2.5"
 
 	V2_4_1 = "2.4.1"
+	V2_5_1 = "2.4.1"
```

Comment:
Should the value of `V2_5_1` be "2.5.1" instead?

Reply:
good catch thanks!